frontend/packages/local-storage-operator-plugin/src/utils/index.ts

```diff
@@ -0,0 +1,6 @@
+import * as _ from 'lodash';
+import { NodeKind } from '@console/internal/module/k8s';
+
+export const hasTaints = (node: NodeKind) => {
+  return !_.isEmpty(node.spec?.taints);
```

Comment:
```suggestion
  return !node.spec?.taints;
```
Would this work?

Reply:
It should be `node.spec?.taints` in order to detect taints which will return list of taints if present, instead of true and false which is not desired here.

Reply:
taints would return you `undefined` if it's untainted else it would return an object. I don't really see the need to use `isEmpty`.

Reply:
https://github.com/openshift/console/pull/5877/files#diff-909ee1940c2521b309b4a7b7d43eab41R46 - here we are accepting a boolean value i.e. is taint there or not and the similar behavior is followed in ocs-install too.
Else we can do `node.spec?.taints ? true: false`

Reply:
I see it's accepting a boolean value, my suggestion should also give you a boolean. You don't need to use lo-dash. Just do `!!node.spec?.taints`.